src/sdk/namespace_fs.js

```diff
@@ -2315,6 +2315,10 @@ class NamespaceFS {
         return xattr[XATTR_MD5_KEY];
     }
 
+    _number_of_tags_fs_xttr(xattr) {
+        return Object.keys(xattr).filter(xattr_key => xattr_key.includes(XATTR_NOOBAA_CUSTOM_PREFIX)).length;
```

Comment:
Can you rename XATTR_NOOBAA_CUSTOM_PREFIX to a name that includes TAG? I find it a very confusing name

Reply:
Do you mean rename `XATTR_NOOBAA_CUSTOM_PREFIX` to `XATTR_NOOBAA_CUSTOM_PREFIX_TAG`?

Reply:
maybe XATTR_TAG, similar to the other xattrs